bamboo/test_suite.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+EXIT_CODE=0
+
+function help_message {
+    local SCRIPT=$(basename ${0})
+    local N=$(tput sgr0)
+    local C=$(tput setf 4)
+    cat <<EOF
+Run test configurations of LBANN.
+By default, run all tests.
+Usage: ${SCRIPT} [options]
+Options:
+    ${C}-h|--help${N}                   Display this help message and exit
+    ${C}-u|--unit${N}                   Run all unit tests
+    ${C}-i|--integration${N}            Run all integration tests 
+    ${C}-c|--compiler${N}               Run all compiler tests
+    ${C}-e|--executable${N} <val>       Specify path to executable (default executable built from build_lbann_lc.sh)
+    ${C}-n|--nobuild${N}                Do not build any executable
+EOF
+}
+
+function print_label {
+    TEST_SET=${1}
+    echo "-----------------------------------------------------------------------------------------------------------------"
+    echo "STARTING ${TEST_SET} TESTS"
+    echo "-----------------------------------------------------------------------------------------------------------------"
+}
+
+function print_results {
+    TEST=${1}
+    if [ $? -ne 0 ]; then
+        EXIT_CODE=1
```

Comment:
This check doesn't seem to work. Bamboo still prints "PASSED" even when the previous lines fail.